constants/constants.go

```diff
@@ -23,16 +23,16 @@ var BoldColor = color.New(color.OpBold).Render
 var LoadingSpinner = spinner.New(spinner.CharSets[9], 100*time.Millisecond, spinner.WithColor("cyan"), spinner.WithHiddenCursor(true))
 
 // RegexDarwin is a regular express for darwin systems
-var RegexDarwin = `(?i)(darwin|mac(os)?|apple|osx)`
+var RegexDarwin = `(?i)(darwin|mac(os)?|apple|osx|.dmg)`
 
 // RegexWindows is a regular express for windows systems
-var RegexWindows = `(?i)(windows|win)`
+var RegexWindows = `(?i)(windows|win|.msi|.exe|.appx)`
 
```

Comment:
Could you remove `.dmg` and `.appx`? Don't think these help because even if you can download them you want be able to properly install these formats through stew.